README.md

```diff
@@ -39,13 +39,15 @@ channel automatically.
 
 1. Select "Start Recording" to start audio recording
 
-1. Post the agenda link in the meeting chat box
+1. Post the link to today's agenda in the meeting chat box. The link below leads 
+   to the default "standing" agenda, and should be posted only if a date-specific 
+   agenda was not sent to the mailing list.
+
+   `Agenda: https://github.com/w3c-ccg/traceability-interop/blob/main/AGENDA.md`
```

Comment:
```suggestion
1. Post the link to today's agenda in the meeting chat box. The link below leads 
   to the default "standing" agenda. If a date-specific agenda was sent to the
   mailing list prior to the meeting, a link to that archived email should be used
   instead.

   `Agenda: https://github.com/w3c-ccg/traceability-interop/blob/main/AGENDA.md`
```
I agree with your suggestion that what we *really* should be doing here is posting the agenda to the mailing list prior to the meeting, and linking to that post _instead_ of using a standing agenda link. I have opened #371 to address that change separately.

I think it is important that we do not suggest that it is ok to omit this item from the meeting chat, as it would prevent the agenda link from showing in the rendered HTML minutes.